subsys/net/lib/sockets/Kconfig

```diff
@@ -32,6 +32,13 @@ config NET_SOCKETS_POLL_MAX
 	help
 	  Maximum number of entries supported for poll() call.
 
+config NET_SOCKETS_SOCKOPT_TLS
+        bool "Enable TCP TLS socket option support"
+	default n
+	help
+	 Enable TCP TLS socket option support which automatically establishes
```

Comment:
According to other kconfig files, there is one extra space indendation to help texts (so should start from l letter in previous help line).

Reply:
Yep, shippable will scream at me.